test/controllers/UpscanControllerSpec.scala

```diff
@@ -82,7 +82,8 @@ class UpscanControllerSpec extends UnitSpec with ErsTestHelper with GuiceOneAppP
       result.header.headers("Location") shouldBe routes.UploadController.uploadCSVFile(Fixtures.getMockSchemeTypeString).toString
     }
 
-    "return a 200 when a file is still in progress" in {
+    //TODO maybe consider showing the user a separate page that tells them it's still in progress instead of a global error page?
```

Comment:
TODO left behind here too.

Reply:
I think it's worth it leaving this one there for the future (since it's not in scope of the ticket) - atm if a user tries to go to the page (refresh/new tab) when a file is still being processed by upscan, they get redirected to the global error page, which could be quite confusing. it would likely not be too difficult to direct them to a new 'in-progress' page instead (since it's already explicitly handling the InProgress situation now anyway)

Reply:
unless you think it's enough to just get a ticket raised and get rid of the todo for cleanliness - happy with that too

Reply:
Yeah, I think this was more of an idea to make the service better for discussion with the design team? I'd be in favour of raising a ticket or chatting with Mark about it over leaving a TODO in the tests